src/IxMilia.Dxf/IxMilia.Dxf.csproj

```diff
@@ -6,7 +6,7 @@
     <AssemblyTitle>IxMilia.Dxf</AssemblyTitle>
     <VersionPrefix>0.7.0</VersionPrefix>
     <Authors>IxMilia</Authors>
-    <TargetFrameworks Condition="'$(OS)' != 'Unix'">netstandard1.0;net45</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Unix'">netstandard1.0;net45;net35</TargetFrameworks>
```

Comment:
Not all machines have the 3.5 SDK installed.  I'd prefer a separate `IxMilia.Dxf.Net35.csproj` exist so that computers without the 3.5 SDK (e.g., mine) can still build as expected.  This might even be a compatibility requirement as older versions of Visual Studio won't be able to open this style of project (with `Sdk="Microsoft.NET.Sdk"` at the project root.)